src/arr/compiler/anf-loop-compiler.arr

```diff
@@ -101,6 +101,8 @@ j-for = J.j-for
 j-raw-code = J.j-raw-code
 make-label-sequence = J.make-label-sequence
 
+var cases-dispatches = cl-empty
```

Comment:
So this value must be accessible from `compile-fun-body`, `compile-split-cases`, and `compile-module`. If we were to initialize it in `compile-module`, we would need to juggling this value everywhere. Having it as the global variable gets rid of this problem.

And it's easy to reset even in the global variable version. Right after we output the bindings, reset it to empty list.

Of course, I'm not saying that it's the best practice. If you think that it's objectively better to instantiate in `compile-module`, I will do that.